src/sentry/static/sentry/app/routes.jsx

```diff
@@ -876,19 +870,46 @@ function routes() {
           <Route
             path="releases/"
             componentPromise={() =>
-              import(/*webpackChunkName:"ProjectReleases"*/ './views/releases/projectReleases')}
+              import(/*webpackChunkName:"ProjectReleases"*/ './views/releases/list/projectReleases')}
```

Comment:
Is this extra `list/details` depth necessary?

Reply:
Doesn't the shared code eventually go away?

Reply:
You mean if/when we remove the single project option (ie. projectReleases) in the UI?